### PR TITLE
2.6.3 release notes

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

**Comment by nathan-px** [comment]
@vilasd I removed duplicates and a torpedo issue:
* 17725
* 17953
* 17477

File: content/reference/release-notes/portworx.md [hunk 1]
```diff
@@ -6,6 +6,34 @@ keywords: portworx, release notes
...
+| PWX-17699 | Portworx created the incorrect type of vSphere disks when using Portworx disk provisioning. <br/><br/>**User impact:** Portworx incorrectly parsed the `lazyzeroedthick` disk type provided by users in the vSphere cloud drives spec, and instead created the default `eagerzeroedthick` disks. <br/><br/>**Resolution:** Portworx now correctly parses the spec and created thes correct disk type. |
+| PWX-17217 | Rel note this |
+| PWX-17450 | Volume mount operations inside pods on destination clusters sometimes failed after async DR/Migration. <br/><br/>**User impact:** Users sometimes needed to restart their pods after DR/Migration to correct failed volume mounts. <br/><br/>**Resolution:** These volume mount operations no longer fail. |
+| PWX-18116 | This commit updates the range of ubuntu azure kernels that require a btrfs patch ???|
```
Comment (nathan-px) [inline content/reference/release-notes/portworx.md:32]: @vilasd @vjs-px Does this need a RN?

File: content/reference/release-notes/portworx.md [hunk 2]
```diff
@@ -6,6 +6,34 @@ keywords: portworx, release notes
...
+| PWX-17155 | Sharedv4 volume consumers were tracked based on nodeID, which changes when a storageless node becomes storage node. <br/><br/>**User impact:** The sharedv4 pods running on a storageless lost access to the mounted sharedv4 volume ( access denied) when that node that was restarted to assume role of storage node. <br/><br/>**Resolution:** Use nodeIP instead of nodeID to track the sharedv4 clients(consumers) as the node IP remains the same when the role changes from storageless to storage node. |
+| PWX-17699 | Portworx created the incorrect type of vSphere disks when using Portworx disk provisioning. <br/><br/>**User impact:** Portworx incorrectly parsed the `lazyzeroedthick` disk type provided by users in the vSphere cloud drives spec, and instead created the default `eagerzeroedthick` disks. <br/><br/>**Resolution:** Portworx now correctly parses the spec and created thes correct disk type. |
+| PWX-17217 | Rel note this |
```
Comment (nathan-px) [inline content/reference/release-notes/portworx.md:30]: @prabirpaul What's the RN for it?

Comment (vilasd) [reply content/reference/release-notes/portworx.md:30]: Nope we don't need to release note this

File: content/reference/release-notes/portworx.md (hunk 1 above)
Comment (vilasd) [reply content/reference/release-notes/portworx.md:32]: No don't need to I believe as it was already done as part of 2.6.2.1

File: content/reference/release-notes/portworx.md [hunk 3]
```diff
@@ -6,6 +6,34 @@ keywords: portworx, release notes
...
+| PWX-17450 | Volume mount operations inside pods on destination clusters sometimes failed after async DR/Migration. <br/><br/>**User impact:** Users sometimes needed to restart their pods after DR/Migration to correct failed volume mounts. <br/><br/>**Resolution:** These volume mount operations no longer fail. |
+| PWX-18116 | This commit updates the range of ubuntu azure kernels that require a btrfs patch ???| 
+| PWX-18071	| ??? | 
+| PSG-184 | ??? |
+| PWX-15700 | ??? |
```
Comment (nathan-px) [inline content/reference/release-notes/portworx.md:34]: @zoxpx Does this need a release note? It was a doc fix right? so probably not.

File: content/reference/release-notes/portworx.md [hunk 4]
```diff
@@ -6,6 +6,33 @@ keywords: portworx, release notes
...
+| PWX-17699 | Portworx created the incorrect type of vSphere disks when using Portworx disk provisioning. <br/><br/>**User impact:** Portworx incorrectly parsed the `lazyzeroedthick` disk type provided by users in the vSphere cloud drives spec, and instead created the default `eagerzeroedthick` disks. <br/><br/>**Resolution:** Portworx now correctly parses the spec and created thes correct disk type. |
+| PWX-17217 | Rel note this |
+| PWX-17450 | Volume mount operations inside pods on destination clusters sometimes failed after async DR/Migration. <br/><br/>**User impact:** Users sometimes needed to restart their pods after DR/Migration to correct failed volume mounts. <br/><br/>**Resolution:** These volume mount operations no longer fail. |
+| PWX-18116 | This commit updates the range of ubuntu azure kernels that require a btrfs patch ???| 
+| PSG-184 | ??? |
```
Comment (nathan-px) [inline content/reference/release-notes/portworx.md:33]: @harsh-px Just double checking. This doesn't need to be in here, does it?

Comment (vilasd) [inline content/reference/release-notes/portworx.md:33]: No need to release note this as we did not fix anything here

File: content/reference/release-notes/portworx.md (hunk 3 above)
Comment (vilasd) [reply content/reference/release-notes/portworx.md:34]: No release note as its doc fix.

**Comment by vilasd** [comment]
@nathan-px https://portworx.atlassian.net/browse/PWX-16412 add this bug to RL note

**Review by vilasd — APPROVED** [review]

**Pull request merged**